test/contract.js

```diff
@@ -0,0 +1,109 @@
+import 'babel-polyfill';
+import { assert } from 'chai';
+const _ = require('lodash');
+
```

Comment:
What about keep all as "import"? import _ from 'lodash';

Reply:
changed all const imports to `import ...`